yb-voyager/cmd/endCommand.go

```diff
@@ -0,0 +1,15 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var endCmd = &cobra.Command{
+	Use:   "end",
+	Short: "This command will mark the end of the migration",
+	Long:  `This command will mark the end of the migration`,
```

Comment:
`Parent of the command that ends the migration and clears all the migration state.`